src/test/java/com/google/impactdashboard/database_manager/DataReadManagerTest.java

```diff
@@ -149,4 +149,12 @@ public void testNoBindingsReturnedForNonexistentProject() {
 
     assertEquals(expected, actual);
   } 
+
+  @Test
+  public void testCorrectMaxTimestamp() {
```

Comment:
please also add a test case for no entries in the table.

Reply:
Added!